lib/publiccloud/basetest.pm

```diff
@@ -58,14 +59,29 @@ sub provider_factory {
 
     }
     elsif ($args{provider} eq 'AZURE') {
-        $provider = publiccloud::azure->new(
-            key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
-            key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
-            region => get_var('PUBLIC_CLOUD_REGION', 'westeurope'),
-            tenantid => get_var('PUBLIC_CLOUD_AZURE_TENANT_ID'),
-            subscription => get_var('PUBLIC_CLOUD_AZURE_SUBSCRIPTION_ID'),
-            username => get_var('PUBLIC_CLOUD_USER', 'azureuser')
-        );
+        $args{service} //= 'AVM';
+        if ($args{service} eq 'ACR') {
+            $provider = publiccloud::acr->new(
+                key_id => get_var('PUBLIC_CLOUD_KEY_ID'),
+                key_secret => get_var('PUBLIC_CLOUD_KEY_SECRET'),
+                region => get_var('PUBLIC_CLOUD_REGION', 'westeurope'),
+                tenantid => get_var('PUBLIC_CLOUD_TENANT_ID'),
```

Comment:
the drop of `AZURE` in variable name , was it intentional ?

Reply:
oh wait, what did happen here?

Reply:
I fixed this and I relaunched the tests to be sure

Reply:
all in green